CMakeLists.txt

```diff
@@ -7,10 +7,16 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 
-include(GetGitRevisionDescription)
-git_describe(PROJECT_VERSION --tags)
-if(NOT PROJECT_VERSION)
-	set(PROJECT_VERSION "1.2+")
+if(NOT PERFORMOUS_VERSION)
+    message("No project version specified. Determining latest tag..")
+    include(GetGitRevisionDescription)
+    git_describe(PROJECT_VERSION --tags)
+    if(NOT PROJECT_VERSION)
+        message("No project version found. Falling back onto default versioning strategy.")
```

Comment:
in which situation are we supposed to pass here? Shouldn't it be an error?

Reply:
If there's no git installed. This was there already but i agree it's kinda weird

Reply:
does anyone still use it? maybe time for cleanup?

Reply:
Well for instance, people could download from github using a zip. Extracting the zip and trying to build results now into a valid version. Whereas it then leads to an error